img_merger.py

```diff
@@ -0,0 +1,110 @@
+import cv2
+import json
+import argparse
+from pathlib import Path
+
+from f2b import F2B, draw
+from coco_utils import *
+
+
+parser = argparse.ArgumentParser()
```

Comment:
[same](https://github.com/levan92/f2b/pull/2/files#r503142777) for this, maybe let's directly give filepaths instead